src/Compilers/VisualBasic/Portable/BoundTree/BoundCall.vb

```diff
@@ -7,6 +7,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
     Partial Friend Class BoundCall
 
+        Public Sub New(
+            syntax As SyntaxNode,
+            method As MethodSymbol,
+            methodGroupOpt As BoundMethodGroup,
+            receiverOpt As BoundExpression,
+            arguments As ImmutableArray(Of BoundExpression),
+            defaultArguments As BitVector,
```

Comment:
>defaultArguments As BitVector, [](start = 12, length = 30)

Why not simply add it as optional to the constructor below?

Reply:
Adding as non optional is even better, this way we will make sure that all call sites properly preserve information

---
In reply to: [190017640](https://github.com/dotnet/roslyn/pull/26781#discussion_r190017640) [](ancestors = 190017640)

Reply:
Adding as optional to the constructor below, as we have 69 callsites that don't pass DefaultArguments currently.

---
In reply to: [190018044](https://github.com/dotnet/roslyn/pull/26781#discussion_r190018044) [](ancestors = 190018044,190017640)